website/project/model.py

```diff
@@ -75,16 +74,9 @@ def has_anonymous_link(node, auth):
     :param str link: any view-only link in the current url
     :return bool anonymous: Whether the node is anonymous to the user or not
     """
-    view_only_link = auth.private_key or request.args.get('view_only', '').strip('/')
```

Comment:
I'm comprehending how the framework works. Looks like we have view functions that are mapped to URLs in `website.routes`, and the view functions are decorated with functions that instantiate `Auth`.